### PR TITLE
MCO-871 MCO add OCL dedicated job

### PR DESCRIPTION
**Pull request by sergiordlr**
Add a new prow job with OCL enabled to run MCO test cases compatible with OCL.

### CODE REVIEW

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-longrun-mco-ocl-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-longrun-mco-ocl-f9

**Comment by sergiordlr** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-longrun-mco-tp-ocl-f7

File: ci-operator/step-registry/mco/conf/day2/enable-ocl/mco-conf-day2-enable-ocl-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        A space splitted array with the pools that will be configured with OCL
+    - name: MCO_CONF_DAY2_OCL_IMG_EXPIRATION_TIME
+      default: "2h"
+      documentation: |
+        The time after which the images will be automatically deleted from the quay repository
+  documentation: |-
+    Configures OCL in the MachineConfigPools provided in the MCO_CONF_DAY2_OCL_POOLS list.
+    The secret used to pull and pull the images will be the cluster's pull-secret.
+    The repository used to store the images will be quay.io/mcoqe/layering
+    All images will be labeled with
```
Comment (ptalgulk01) [inline ci-operator/step-registry/mco/conf/day2/enable-ocl/mco-conf-day2-enable-ocl-ref.yaml:25]: The last line is incomplete here

**Pull request merged**